### ISSUE TITLE
please contact me

### ISSUE THREAD

**Issue by snchoudhary** [VarunS2002/Python-NSE-Option-Chain-Analyzer#46]
@VarunS2002 Hello brother i see you done great work in option chain analyzer. i want to update you that nse option chain data is in contracts. so it is difficult in analysis. if we multily one lot with qantity size it come more accurate. for example nifty one contract multiply with 50 and banknifty multiply with 25. then it come in lakhs. so we want that if you can add a feature where we can see OI in lakhs. it is more useful. i try to contact you but i don't understand how to reach you. if you see my message then please contact me at telegram username alpha225599. i want to contact you.

**Comment by VarunS2002** [comment]
@snchoudhary 
These type of requests are very common. Everyone has their own preference of the units of the values and different multiples.
This is why I provide the feature to export to a `.csv` file so people can use their own MS Excel scripts and manipulate the data accordingly.

For future purposes: This is where you are supposed to contact me and you can set the title to be the feature request.

**Comment by snchoudhary** [comment]
but the problem is i am unable to see data in live market. If csv file is open then data is not exporting. if csv file closed data is exporting. that's why i am telling you just add feature in only index, like Nifty(multiply 50) And Banknifty(multiply 25) it is great feature. because majority of people see data in lakhs. otherwise you can add feature where select option. Oi in Lakh and Oi in Contracts. 
and one more thing 50 and 25 quantity maybe change in future by nse. like the old nifty qantity is 75. when we multiply by 75.

only problem is we can't see data in live market csv file.

**Comment by VarunS2002** [comment]
@snchoudhary 
You have to import the `.csv` data into a separate Excel file write a macro in Excel to continually refresh the file's data.
In this new file there will be the data which you have to multiply by the preferred value.

Regardless, I will consider adding a custom multiple in the future.

**Comment by VarunS2002** [comment]
> I try to imprort in separate file but not working well. Facing many issues in live data. I will be happy if you add multiply feature in next update. It is very very useful for all. I hope you will do. Thank you. You are great person.

Yes I am aware there are some issues which need workarounds that's why I will be considering this feature in the future.

**Comment by VarunS2002** [comment]
@snchoudhary Could you please name the columns/labels of the values that have to be multiplied?
